test_files/functions.sickle.ts

```diff
@@ -17,3 +17,7 @@ function FunctionsTest2(a: number, b: number) {}
  * @param { number} b
  */
 function FunctionsTest3(a: number, b: number) {}
+/**
+ * @param {{a:number, b:number}} param0
```

Comment:
Just for more test coverage, you might throw in two destructuring params here, or a destructure of the form

function foo({a, b}, [c, d])